src/main/java/org/spongepowered/mod/mixin/core/common/MixinSpongeImplHooks.java

```diff
@@ -286,4 +293,23 @@ public static void onTileEntityError(TileEntity tileEntity, CrashReport crashRep
     public static void blockExploded(Block block, World world, BlockPos blockpos, Explosion explosion) {
         block.onBlockExploded(world, blockpos, explosion);
     }
+
+    // Crafting
+
+    @Overwrite
```

Comment:
Oh, this is `SpongeImplHooks`... Nevermind, the purpose of all the overwrites in this class is clear, so I don't think we need them in this case.